apps/expo/src/components/muscleGroup/FrontBackSwitch.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useRef, useState } from 'react'
+import {
+  Animated,
+  Dimensions,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+
+// screen dimensions
+// Scaled based on screenWidth and screenHeight
+const { width: screenWidth, height: screenHeight } = Dimensions.get('window')
+
+// Custom switch component show at bottom - With Animation
+// Call as follows:
+// <onValueChange
+//     label="..."
+//     value={useState()}
+//     onValueChange={() => set(!On)}
+// />
+export default function FrontBackSwitch ({
+  value,
+  onValueChange,
+  label,
+}: {
+  value: boolean
+  onValueChange: () => void
+  label: string
+})  {
+  // Use useRef to persist the animated value without reinitializing it on every render
```

Comment:
make sure to document AI usage

Reply:
Where should I add it?
In this PR page?

Reply:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/43ab964b-f766-449d-8edf-3898996819f7)

At the end of this pr documentation, add if you use any

Reply:
Thanks